node-installer/Makefile

```diff
@@ -19,6 +34,27 @@ build-multi-installer-image: $(BINARY)
 build-dev-installer-image: $(BINARY)
 	docker buildx build -t $(IMAGE_NAME) --load --platform $(PLATFORM) .
 
+deploy-workload:
```

Comment:
Is there a reason you chose to put these in a makefile rather than in another script that can be imported?

Reply:
Nope, it's just easier for me to write...